frontend/src/global.d.ts

```diff
@@ -0,0 +1,6 @@
+import { ExternalProvider } from 'ethersproject/providers';
```

Comment:
It is needed to type window.ethereum which is used in order to prevent metamask from popping up when it is unlocked. I tried to originally include this in the vue-shims file, but it ended up causing build errors. I don't really know exactly why but splitting it to a separate file seemed to fix it.